driver-core/src/main/java/com/datastax/driver/core/QueryOptions.java

```diff
@@ -42,10 +42,32 @@
      */
     public static final boolean DEFAULT_IDEMPOTENCE = false;
 
+    public static final int DEFAULT_MAX_PENDING_REFRESH_NODE_LIST_REQUESTS = 20;
+
+    public static final int DEFAULT_MAX_PENDING_REFRESH_NODE_REQUESTS = 20;
+
+    public static final int DEFAULT_MAX_PENDING_REFRESH_SCHEMA_REQUESTS = 20;
+
+    public static final int DEFAULT_REFRESH_NODE_LIST_INTERVAL_MILLIS = 1000;
+
+    public static final int DEFAULT_REFRESH_NODE_INTERVAL_MILLIS = 1000;
+
+    public static final int DEFAULT_REFRESH_SCHEMA_INTERVAL_MILLIS = 1000;
```

Comment:
I think debouncing is mainly intended for when you don't do things responsibly :)

But I agree that the default are open for discussion, we should discuss them once we've run a few more tests.